.cicd/defaults.json

```diff
@@ -4,6 +4,6 @@
        "prerelease":true
     },
     "referencecontracts":{
-       "ref":"main"
+       "ref":"fix_failures_by_default_tester"
```

Comment:
Remember to change back after merge.

Reply:
Done by https://github.com/AntelopeIO/spring/pull/233